conda_build/conda_interface.py

```diff
@@ -22,7 +22,7 @@
 from conda.signature import KEYS, KEYS_DIR, hash_file, verify  # NOQA
 from conda.utils import human_bytes, hashsum_file, md5_file, memoized, unix_path_to_win, win_path_to_unix, url_path  # NOQA
 import conda.config as cc  # NOQA
-from conda.config import sys_rc_path  # NOQA
+from conda.config import rc_path  # NOQA
```

Comment:
Both 4.1 and 4.2 have `rc_path` and `sys_rc_path`. In both `rc_path` is populated by calling `get_rc_path()` which looks for all possible condarc files and returns the first one that exists.